source/tutorials/migrate-time-series.txt

```diff
@@ -83,23 +69,9 @@ Migrate a Collection to a Time Series Collection
       Create an interactive shell session on the tutorial Docker container
```

Comment:
I feel like it should be more clear that you should do this in another shell window because the time series data is running in the first one from Step 2 Generate Sample Data

Reply:
Added "in a seperate shell window" here